cypress/e2e/article.cy.js

```diff
@@ -1,13 +1,51 @@
+/// <reference types='cypress' />
+
 describe('', () => {
```

Comment:
The describe block should have a suitable description about the set of tests it contains.